scripts/_lines_of_code.py

```diff
@@ -45,17 +45,28 @@ def execute_bind(id_, message):
 class NotContributedError(Exception):
     pass
 
-
+excepted_files = [
+    'package.json',
+    'package-lock.json',
+    '.gitignore',
+    'yarn.lock'
+]
 class ContributedRepository:
     def __init__(self, contributed_to: Repository, user):
         self.total_lines_of_addition_in_contribution = 0
         self.total_lines_of_deletion_in_contribution = 0
         self.name = contributed_to.full_name
-        for contribution in contributed_to.get_stats_contributors():
-            if contribution.author.id == user.id:
-                for week in contribution.weeks:
-                    self.total_lines_of_addition_in_contribution += week.a
-                    self.total_lines_of_deletion_in_contribution += week.d
+
+        if contributed_to.get_commits().totalCount >= 100:
+            execute_bind('<inform>', 'This repository has a lot of commits! This can take some time...')
+
+        for commit in contributed_to.get_commits():
+            if commit.author.id == user.id:
+                for file in commit.files:
+                    if not file.filename in excepted_files or file.filename.startswith('node_modules/'):
+                        self.total_lines_of_addition_in_contribution += file.additions
+                        self.total_lines_of_addition_in_contribution += file.deletions
+
```

Comment:
```commit.author``` is defined as None when it is not commited by a Github user (likely commited via workflow). we can just ignore them (by using ```if not commit.author is None```).
```suggestion
        for commit in contributed_to.get_commits():
            if commit.author is not None:
                if commit.author.id == user.id:
                    for file in commit.files:
                        if not file.filename in excepted_files or file.filename.startswith('node_modules/'):
                            self.total_lines_of_addition_in_contribution += file.additions
                            self.total_lines_of_addition_in_contribution += file.deletions

```